src/hooks/useChainHelpers.ts

```diff
@@ -216,6 +230,24 @@ export default function useChainHelpers() {
         nativeCurrency: { decimals: 18, name: "SGB", symbol: "SGB" }, 
         rpcUrls: ["https://songbird.towolabs.com/rpc", "https://songbird-api.flare.network/ext/C/rpc"]
       }] 
+    } else if (networkName == "Coston Testnet") {
+      method = "wallet_addEthereumChain"
+      params = [{ 
+        blockExplorerUrls: [ "https://coston-explorer.flare.network" ],
+        chainId: "0x10",
+        chainName: "Coston Testnet",
+        nativeCurrency: { decimals: 18, name: "FLR", symbol: "FLR" }, 
+        rpcUrls: ["https://coston-api.flare.network/ext/C/rpc"]
+      }] 
+    } else if (networkName == "Flare") {
+      method = "wallet_addEthereumChain"
+      params = [{ 
+        blockExplorerUrls: [ "https://flare-explorer.flare.network" ],
+        chainId: "0x0e",
```

Comment:
@chnejohnson I think it's uppercase E. Try just `0xE`, but if it doesn't work try `0x0E`.